src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx

```diff
@@ -10,15 +10,75 @@ import { FormControl, FormLabel, Grid, MenuItem, TextField } from '@mui/material
 import ReactHookTextField from '../../../components/ReactHookTextField';
 import { Control, Controller, FieldErrorsImpl } from 'react-hook-form';
 import { DatePicker } from '@mui/x-date-pickers';
-import { WorkPackageEditFormPayload } from './WorkPackageEditContainer';
+import NERAutocomplete from '../../../components/NERAutocomplete';
 
 interface Props {
-  users: User[];
-  control: Control<WorkPackageEditFormPayload>;
-  errors: Partial<FieldErrorsImpl<WorkPackageEditFormPayload>>;
+  lead?: string;
+  manager?: string;
+  setManager: (manager?: string) => void;
+  setLead: (lead?: string) => void;
+  usersForProjectLead: User[];
+  usersForProjectManager: User[];
+  control: Control<
+    {
+      workPackageId: number;
```

Comment:
I think you can just use the type that was here before (WorkPackageEditFormPayload) instead of redefining it like this. If you do end up having to redefine it tho, like separately define it and then call it here (same for errors)